application.py

```diff
@@ -2,9 +2,12 @@
 from __future__ import print_function
 
 from flask import Flask
+from werkzeug.serving import WSGIRequestHandler
 
 from app import create_app
 
+WSGIRequestHandler.version_string = lambda self: "SecureServer"
```

Comment:
This seems like it is tapping into an undocumented property after looking over the documentation; is that the case, and does it only accept a function as an assignment, hence the `lambda` statement?

Since the finding is related to the `Server` HTTP response header, could we fix/override the information with the header directly upon returning the response, just like with the other response headers above.  Could we try that approach instead and see if it works?

Reply:
Gotcha, thanks for digging into that to check, I appreciate it!  Let's leave as is then so that it's there and addresses the warning instead of ignoring it.